src/components/Layout/RootLayout/index.tsx

```diff
@@ -1,44 +1,61 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import { Outlet } from 'react-router-dom';
-
-import { assets } from '@/assets/assets';
-import theme from '@/theme';
+import { AppProvider } from '@toolpad/core/react-router-dom';
 import ListIcon from '@mui/icons-material/List';
 import LocationOnIcon from '@mui/icons-material/LocationOn';
 import LuggageIcon from '@mui/icons-material/Luggage';
 import SettingsIcon from '@mui/icons-material/Settings';
-import { AppProvider } from '@toolpad/core/react-router-dom';
+
+import { assets } from '@/assets/assets';
+import theme from '@/theme';
+import { RootState } from '@/store/store';
+import CustomNavigationItem from '@/components/NavigationCount';
+import { useRootLayout } from './useRootLayout';
 
 import type { Navigation } from '@toolpad/core';
+
 const RootLayout: React.FC = () => {
   const { t } = useTranslation();
+  const { role } = useSelector((state: RootState) => state.auth);
+  const { newOrdersCount } = useRootLayout();
 
-  const NAVIGATION: Navigation = React.useMemo(
-    () => [
-      {
-        segment: 'orders',
-        title: t('navigation.orders'),
-        icon: <LuggageIcon />,
-      },
-      {
-        segment: 'routes',
-        title: t('navigation.routes'),
-        icon: <LocationOnIcon />,
-      },
-      {
-        segment: 'settings',
-        title: t('navigation.settings'),
-        icon: <SettingsIcon />,
-      },
-      {
-        segment: 'company-list',
-        title: t('navigation.companyList'),
-        icon: <ListIcon />,
-      },
-    ],
-    [t]
-  );
+  const NAVIGATION: Navigation = React.useMemo(() => {
+    if (role === 'superadmin') {
```

Comment:
1.1.5
roles should be stored in enums in the general constants

Reply:
done